tecken/tests/test_sentry.py

```diff
@@ -221,8 +220,9 @@ def test_sentry_scrubbing(sentry_helper, transactional_db):
         assert differences == []
 
 
-def test_count_sentry_scrub_error():
-    with MetricsMock() as metricsmock:
```

Comment:
Weird. I wonder why I used the contextmanager mode rather than the pytest fixture. Thank you for fixing this!